### PR TITLE
fix duplicate main error for android #107

### PR DESCRIPTION
**Pull request by waruqi**
https://github.com/libbpf/libbpf-bootstrap/issues/107

### CODE REVIEW

**Comment by anakryiko** [comment]
minimal_legacy is a separate binary, let's define a separate target for it instead of just dropping it?

Also, please make sure your commit message has description. And it would be nice to add your Signed-off-by, like other commits do (that's kernel and libbpf style, so would be nice to stay consistent): https://github.com/libbpf/libbpf-bootstrap/commit/599e9ac6ad0947838e18ef606076fe66345f498f

**Comment by waruqi** [comment]
> And it would be nice to add your Signed-off-by

This also requires configuring pgp signatures, but my previous signature environment was corrupted, but I didn't have much time to work out how to fix it.

**Comment by anakryiko** [comment]
> > And it would be nice to add your Signed-off-by
> 
> This also requires configuring pgp signatures, but my previous signature environment was corrupted, but I didn't have much time to work out how to fix it.

it shouldn't it's just a piece of text in your commit message, just spell out your full name and email. E.g., for me it's just adding this text line in commit:

Signed-off-by: Andrii Nakryiko <andrii@kernel.org>

I didn't mean to ask you for a heavy-weight PGP setup.

File: examples/c/xmake.lua [hunk 1]
```diff
@@ -54,7 +54,15 @@ end
...
-    add_files("minimal*.c")
+    add_files("minimal.c", "minimal.bpf.c")
+    add_packages("linux-headers")
+    if not has_config("system-libbpf") then
+        add_deps("libbpf")
+    end
+
+target("minimal_legacy")
+    set_kind("binary")
+    add_files("minimal_legacy*.c")
```
Comment (anakryiko) [inline examples/c/xmake.lua:65]: add_files("minimal_legacy.c", "minimal_legacy.bpf.c")

consistent with other definitions

**Comment by anakryiko** [comment]
Thanks!

**Pull request merged**